plugins/dist_table_procedure.h

```diff
@@ -185,10 +185,12 @@ struct DistTableProcedure: public Procedure {
 
     DistTableProcedureConfig procConfig;
 
-    static void persist(const Url & modelFileUrl, DistTableMode mode, 
+    static void persist(const Url & modelFileUrl, DistTableMode mode,
                         const DistTablesMap & distTablesMap);
 
-    static const int DIST_TABLE_PERSIST_VERSION = 2;
```

Comment:
You just need static constexpr int DIST_TABLE_PERSIST_VERSION = 2;

For older versions of GCC, sometimes you need enum { DIST_TABLE_PERSIST_VERSION=2 };